mffpy/xml_files.py

```diff
@@ -65,14 +65,17 @@ def register(typ, xml_type):
             return False
 
     @classmethod
-    def from_file(typ, filepointer: FilePointer):
+    def from_file(typ, filepointer: FilePointer, recover=True):
         """return new `XMLType` instance of the appropriate sub-class
 
         **Parameters**
         *filepointer*: str or IO[bytes]
             pointer to the xml file
+        *recover*: bool
+            indicates whether to try hard to parse through broken XML or not
```

Comment:
Can you document here that we set this to true by default because it's necessary if there are weird characters in the xml file, which can occasionally occur?